app.js

```diff
@@ -25,4 +25,164 @@ window.addEventListener('DOMContentLoaded', () => {
+      source: 'https://github.com/ichsansandy/portfolio-microverse',
+      live: 'https://github.com/ichsansandy/portfolio-microverse',
+    },
+  ];
+
+  const cards = () => {
+    const card = portfolioList.map(
+      (portfolio) => `<div class="card">
+      <div class="card-image" style="--image:url('${portfolio.screenshotMobile}')"></div>
+      <div class="card-info">
+      <h4 class="card-title title-text center-text dark">${portfolio.title}</h4>
+      <ul class="tag-container">
+      <li class="tag flex-center">${portfolio.techStack[0]}</li>
+      <li class="tag flex-center">${portfolio.techStack[1]}</li>
+      <li class="tag flex-center">${portfolio.techStack[2]}</li>
+      <li class="tag flex-center">${portfolio.techStack[3]}</li>
+      </ul>
+      <button class="button card-button white modal-button" data-modal="modal${portfolio.id}">See project</button>
+    </div>
+    </div>
+    <dialog id="modal${portfolio.id}">
+    <div class="modal">
+    <button class="close-modal-button" data-modal="modal${portfolio.id}"><i class="fas fa-xmark fa-lg"></i></button>
+    <div class="modal-image" style="--imageMobile:url('${portfolio.screenshotMobile}'); --imageDekstop:url('${portfolio.screenshotDekstop}')" width="100%" height="220px"></div>
+    <h3 class="modal-title">${portfolio.title}</h3>
+    <ul class="tag-container">
+    <li class="tag flex-center">${portfolio.techStack[0]}</li>
+    <li class="tag flex-center">${portfolio.techStack[1]}</li>
+    <li class="tag flex-center">${portfolio.techStack[2]}</li>
+    </ul>
+    <p>${portfolio.details}</p>
+    <p>${portfolio.details}</p>
+    <div class="modal-button-container">
+    <a class="button card-button modal-link" href=${portfolio.live} target="_blank">
+    See live <span><i class="fa-duotone fa-arrow-up-left-from-circle fa-rotate-90 fa-xl fa-fw"></i></span>
+    </a>
+    <a class="button card-button modal-link" href=${portfolio.source} target="_blank">
+    See source <span><i class="fa-brands fa-github fa-xl fa-fw"></i></span>
+    </a>
+    </div>
+    </div>
+    </dialog>
+    `,
+    );
+    return card;
+  };
+
+  document.querySelector('.card-container').innerHTML = cards().join('');
+
+  const modalButtons = document.querySelectorAll('.modal-button');
+  const closeButtons = document.querySelectorAll('.close-modal-button');
+
+  if (modalButtons) {
+    // eslint-disable-next-line no-plusplus
+    for (let index = 0; index < modalButtons.length; index++) {
+      const element = modalButtons[index];
+      const modalId = element.getAttribute('data-modal');
+      const modal = document.getElementById(`${modalId}`);
+
+      element.addEventListener('click', () => {
+        modal.showModal();
+        body.classList.add('prevent-scrolling');
```

Comment:
- [ ] Kindly make sure the image of the project is the one that pops up on the modal once the pop-up is active. As you may notice when the user clicks on the one you implemented they get a whole different image and not the one of the project.please fix that for a better user experience.

Reply:
Hi @christianbiring1 
Thank you for the review
I didn't really understand this one, could you elaborate?
If you mean about the difference of the image of desktop and mobile version because I am following the template
Or I should only use one image

Currently only the first card has image and the other didn't have any image

Reply:
@ichsansandy , What I meant is that, when the user clicks on a specific project (the ones that have images) that image is also the one that supposed to be on the pop-up model of that specific project as you did for the project `title`, `description` ...

Reply:
Ok ,got it thanks